.eslintrc

```diff
@@ -0,0 +1,33 @@
+{
+    "root": true,
+  "extends": "airbnb-base",
+  "env": {
+    "node": true,
+    "es6": true,
+    "mocha": true
+  },
+  "rules": {
+    "one-var": 0,
```

Comment:
one-var should not be turned off

Reply:
So you actually do not need to define it

Reply:
Thank you for your feedback. This has been corrected

Reply:
Thank you for your feedback. I am on it